docs/updates/migrate-v3.md

```diff
@@ -16,18 +16,30 @@ Going forward, **only projects with Specification Versions 2.0 and higher are su
 In case a legacy specification version is detected, **an automatic migration is attempted.**
 This means your old projects might still work. Unless they have non-standard configuration in their ui5.yaml.
 
+## Changes for Projects
+
+!!! info
+    ✅ Projects defining **Specification Version 2.x** are expected to be **fully compatible with UI5 Tooling v3**
+
+For projects defining the latest **Specification Versions 3.0 and higher**, some changes apply:
+
+* **Breaking Change:** The `metadata.name` property is now restricted to contain only certain characters and no uppercase letters. See [Configuration: `name`](../pages/Configuration.md#name) for details
+
+Also see [Configuration: Specification Version 3.0](../pages/Configuration.md#specification-version-30-beta).
+
```

Comment:
We could remove the "beta" from the section name in Configuration ("### Specification Version 3.0 [beta]"), also given that there's still a development disclaimer for specVersion 3.0. This would allow to use the future-proof link "../pages/Configuration.md#specification-version-30" without the beta already.

Reply:
I'll do that in a follow up change